### ISSUE TITLE
ViewPager fragment recyclerview scroll not working

### ISSUE THREAD

**Issue by hardikamal** [miguelhincapie/CustomBottomSheetBehavior#49]
Hi,

I have added viewpager and fragment has recyclerview in it. On scrolling down, the bottom sheet gets scrolled instead of recyclerview...

**Comment by vijayinnovify** [comment]
Same issue here

**Comment by vjayrajput** [comment]
hi any one find solution for it

https://github.com/laenger/ViewPagerBottomSheet

**Comment by miguelhincapie** [comment]
Hi guys,
I'm gonna try adding the behavior that @vjayrajput suggest [https://github.com/laenger/ViewPagerBottomSheet](https://github.com/laenger/ViewPagerBottomSheet)

**Comment by jhass** [comment]
@miguelhincapie any updates regarding that?

**Comment by miguelhincapie** [comment]
Nope, not enough time to work in this project 😅 . I saw the desired behavior and is easily to implement.
If someone wants to give a hand with this, go ahead! help the community :)
